rollingpin/main.py

```diff
@@ -86,11 +87,12 @@ def load_provider(provider_type, config_parser):
     return provider_cls(provider_config)
 
 
-def _load_configuration():
+def _load_configuration(profile_name, profile_directory="/etc/rollingpin.d/"):
```

Comment:
Derp, yeah... I had a few iterations on this and testing it and the double definition is definitely an artifact of it... thanks, will change this.